vyper/codegen/stmt.py

```diff
@@ -284,10 +276,12 @@ def _parse_For_range(self):
 
         self.context.forvars[varname] = True
 
-        loop_body = ["seq"]
-        # store the current value of i so it is accessible to userland
-        loop_body.append(["mstore", iptr, i])
-        loop_body.append(parse_body(self.stmt.body, self.context))
+        loop_body = [
+            "seq",
+            # store the current value of i, so it is accessible to userland
+            ["mstore", iptr, i],
+            parse_body(self.stmt.body, self.context),
+        ]
```

Comment:
let's revert this change to the previous style please, this style ends up with too much nesting

Reply:
I do like to strive for immutability, but I'm fine with reverting this